payment.yaml

```diff
@@ -0,0 +1,187 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Payment Ecosystem
+  description: Internal API that supplies abstraction over blockchain
+servers:
+  - url: 'https://api.kinmarketplace.internal/v1/'
+paths:
+  /transactions/{transaction_id}:
```

Comment:
to be consistent with Stellar terms, it might be called 'payments' instead. In Stellar, a transaction is account creation, trustline, payment, offer, trade.

Reply:
I changed it to Orders. It seems that both @fuzzyami and us are using this term for internal application transactions